uiServices/homepage.go

```diff
@@ -52,6 +53,35 @@ func getIndexPage() *chi.Mux {
 		w.Header().Set(contentTypeHeader, htmlType)
 		w.Write([]byte(htmlBytes))
 	})
+
+	router.Get("/*", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
this router.Get("/*" needs to be the last router operation in this function otherwise it will conflict with any GETs we add later. Move it below line 85 (and take Frank's changes).